com.unity.ml-agents/Editor/AgentEditor.cs

```diff
@@ -22,6 +22,9 @@ public override void OnInspectorGUI()
                 new GUIContent("Max Step", "The per-agent maximum number of steps.")
             );
 
+            var observableAttributeBehavior = serializedAgent.FindProperty("m_observableAttributeBehavior");
```

Comment:
observableAttributeBehavior -> observableAttributeHandling
Behavior is already used.

Reply:
Done